wormhole-connect/src/components/Modal.tsx

```diff
@@ -26,6 +26,8 @@ const useStyles = makeStyles<{ width: number }>()((theme: any, { width }) => ({
       margin: '20px auto',
       padding: '24px 12px',
     },
+    background: 'rgba(255,255,255,.02)',
```

Comment:
Maybe it's related to this line.  But I noticed an ever so slight difference in color between the inner/outter modal content.  Should all be the same color

Reply:
ah, did that to discern the difference (of when clicking would close vs not), but agree with leaving as is for now. can always revisit.